manifests/charts/istiod-remote/files/injection-template.yaml

```diff
@@ -200,6 +200,12 @@ template: |
       valueFrom:
         fieldRef:
           fieldPath: metadata.labels['service.istio.io/canonical-revision']
+    # To prevent Envoy from unnecessarily trying to run through DNS search domains first
+    # for every DNS query. ndots 0 forces it to resolve the hostname as is first before
```

Comment:
Will this impact DNS clusters as well? I am a bit concerned these will now break, especially since these requests will not be captured by envoy at all?

Reply:
No it wont.. In fact, it will make them faster. Envoy wont try to resolve google.com.ns1.svc.cluster.local, google.com.svc.cluster.local, etc. Instead it will directly resolve google.com.

Reply:
and similarly - when they are relying on the searches it will suddenly behave differently. I would really like the experimental DNS stuff to not impact a stable feature.

Reply:
??? ndots does NOT change the correctness. It merely pushes the literal query name search to the beginning instead of the end. If a hostname resolves only after expanding, it will continue to resolve the same way. Value of ndots will not impact it. 
```sets a threshold for the number of dots which must appear in a name given to res_query(3) (see resolver(3)) before an initial absolute query will be made. The default for n is 1, meaning that if there are any dots in a name, the name will be tried first as an absolute name before any search list elements are appended to it. The value for this option is silently capped to 15.
```

from the resolv conf man page. HTH.